CMakeLists.txt

```diff
@@ -560,9 +605,9 @@ endif()
 list(APPEND SRC ${CMAKE_CURRENT_BINARY_DIR}/PluginManager.cpp)
 
 # directories
-
-include_directories(${INCLUDE})
-link_directories()
```

Comment:
I can't remember removing them, so it was most likely by mistake... I didn’t experience any problems though, probably because the include_directories() line is also called in the demos section.